src/noisepy/seis/noise_module.py

```diff
@@ -523,7 +523,7 @@ def cut_trace_make_stat(fc_para: ConfigParameters, ch_data: ChannelData):
     # trace_madS = np.zeros(nseg,dtype=np.float32)
     trace_stdS = np.zeros(nseg, dtype=np.float32)
     dataS = np.zeros(shape=(int(nseg), int(npts)), dtype=np.float32)
-    dataS_t = np.zeros(nseg, dtype=np.float)
+    dataS_t = np.zeros(nseg, dtype=np.float32)
```

Comment:
@mdenolle `np.float` has been deprecated. I changed this to `np.float32` to be consistent with the other arrays used here. But let me know if this particular one should be `np.float64`.